packages/s3/src/model/files/service.ts

```diff
@@ -14,6 +17,10 @@ class FileService<
   extends BaseService<File, FileCreateInput, FileUpdateInput>
   // eslint-disable-next-line prettier/prettier
   implements Service<File, FileCreateInput, FileUpdateInput> {
+  protected s3Client = new S3Client(this.config);
```

Comment:
this should be encapsulated inside a function

Reply:
@rameshlohala 
Creating an instance within a getter, is generally not a recommended approach because it will create new instance every call of function
it recommended to init it inside a constructor.

Reply:
after creation you have to set it to a protected property so it calling it won't create new instance every time. you can check getter for sql factory in service class in slonik package.